src/components/ProfileSelection.vue

```diff
@@ -39,11 +42,13 @@ export default class ProfileSelection extends Vue {
   async getPresets() {
     const presets = (await ipcRenderer.invoke(
       IPCEvents.GET_PRESETS
-    )) as string[];
+    )) as FriendlyDirectoryMap[];
+
     const userPreset = userPreferences.get(USER_PREFERENCE_KEYS.PRESET);
-    this.presets = presets
-      .map((preset) => ({ text: preset, value: preset }))
-      .filter((preset) => preset.text !== ".DS_Store");
+    this.presets = presets.map((preset) => ({
+      text: preset.friendly,
+      value: preset.real,
+    }));
```

Comment:
We could use object destructuring to clean this up slightly

```suggestion
    this.presets = presets.map(({friendly, real}) => ({
      text: friendly,
      value: real,
    }));
```